### PR TITLE
Added contributer Deven Phillips

### PR DESCRIPTION
**Pull request by InfoSec812**

### CODE REVIEW

File: src/site/community/index.html [hunk 1]
```diff
@@ -104,6 +104,28 @@ <h4 class="media-heading">Michel Kr&auml;mer</h4>
         </div>
       </div>
     </div>
+    <div class="row">
+      <div class="col-xs-12 col-sm-4 col-lg-3">
+        <h3>Contributors</h3>
```
Comment (michel-kraemer) [inline src/site/community/index.html]: Please do not repeat the heading. Just add your column.

File: src/site/community/index.html [hunk 2]
```diff
@@ -104,6 +104,28 @@ <h4 class="media-heading">Michel Kr&auml;mer</h4>
...
+      </div>
+      <div class="col-xs-12 col-sm-4 col-lg-3">
+        <div class="media">
+          <div class="media-left">
+            <img class="contributor-avatar" src="https://en.gravatar.com/userimage/30123434/0b6e8988b6f3ad4e127c8cd0fc9a99c7.jpg?size=80">
+          </div>
+          <div class="media-body">
+            <h4 class="media-heading">Deven Phillips</h4>
+            <p>
+              Web-site<br>
```
Comment (michel-kraemer) [inline src/site/community/index.html]: Is this correct? 'Web-site' should be replaced by your contribution, unless you worked on the web-site of course.

Comment (InfoSec812) [inline src/site/community/index.html]: Whoops, I thought I had put my blog in there! I'll fix it. Gimme a few
minutes.

Deven Phillips
from my mobile

**Comment by InfoSec812** [comment]
Fixed.

**Comment by michel-kraemer** [comment]
I'm sorry, but I need to validate all requests before I can merge them. Would you mind giving me a link to a commit that was successfully merged into one of the official Vert.x repositories? Thank you so much!

By the way, I just changed the community page. The contributors are now in a JSON array and the HTML code is automatically generated in the template. This way we can make sure all entries look the same. I can try to update your pull request, but if you like you can also do it yourself.

Thanks,
Michel

**Comment by InfoSec812** [comment]
Here's one link: https://github.com/vert-x3/vertx-examples/pull/3

**Comment by InfoSec812** [comment]
Some Maven archetypes I created for the project. 

https://github.com/InfoSec812/vertx-js-archetype

 https://github.com/JUGGL/vertx-java-archetype

**Comment by InfoSec812** [comment]
Finally, conference workshop I did on Vert.x 

https://github.com/JUGGL/codepalousa-vertx-chat

**Comment by michel-kraemer** [comment]
Thanks a lot and sorry again for being to picky :-)

**Comment by michel-kraemer** [comment]
Deven,

What do think about your entry on the community page http://vert-x3.github.io/community/ ? I think it looks a bit off due to the long list of contributions, but that's just my personal opinion. Just let me know what you think.

Cheers,
Michel

**Pull request merged**